microsoft-edge/webview2/concepts/user-data-folder.md

```diff
@@ -28,6 +28,7 @@ WebView2 creates the UDF in the default location for the platform, or in the cus
 
 By default, WebView2 creates a UDF in the default location for the particular platform.  This works well on some platforms, but not others.  If your app has specific needs, you can specify a custom UDF location.  Make sure that the custom UDF location that you specify has appropriate Read/Write permissions for the WebView2 app runtime.
 
```

Comment:
I might push a commit to try a section heading that gathers two points.

If we want to highlight the point more, could add an h4 heading is a more low-key alternative to an Alert.  Or, the previous sentence seems like it could be grouped in a list with the present point.

h4: Store user settings on a non-network drive [narrowly focused heading]
or:
h4: Suitable custom UDF locations [useful general heading]

* Make sure that the custom UDF location that you specify has appropriate Read/Write permissions for the WebView2 app runtime.
* Avoid storing user settings on a network drive. This can result in slowdowns, crashes, or loss of data.


@nishitha-burman @jasonstephen15 
https://docs.microsoft.com/en-us/contribute/markdown-reference#alerts-note-tip-important-caution-warning
"Avoid notes, tips, and important boxes. Readers tend to skip over them. It's better to put that info directly into the article text."